src/components/SecondaryPanes/Expressions.js

```diff
@@ -40,13 +41,15 @@ function getValue(expression) {
 }
 
 const Expressions = React.createClass({
+  _input: (null: any),
```

Comment:
I was getting `property '_input'. Property not found in React  component` errors, and I found this [fix](https://github.com/facebook/flow/issues/1676).